### PR TITLE
[Relay][OP] Register topi schedule for Relay fast_exp and fast_tanh

### PR DESCRIPTION
**Pull request by selo1412**
I raised a question in TVM community. 
(https://discuss.tvm.ai/t/relay-op-fast-exp-cant-be-built/6046).
If there's any problem, please don't hesitate to point it out. Thanks.

### CODE REVIEW

**Comment by comaniac** [comment]
Could we have a unit test to guard this issue in the future?

cc @icemelon9

**Comment by icemelon9** [comment]
@selo1412 Could you address @comaniac comments?

**Comment by selo1412** [comment]
@comaniac @icemelon9 
I took 
tvm/tests/python/relay/test_pass_fast_math.py
tvm/topi/tests/python/test_topi_math.py
as reference to add unit test 
tvm/tests/python/relay/test_op_fast_math.py .
and I found some error in topi/tests/python/ make the commit 5 cannot be built.

Is there any suggestions for the above situation?

File: tests/python/relay/test_op_fast_math.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        m.run()
+        # get outputs
+        tvm_output = m.get_output(0)
+        tvm.testing.assert_allclose(tvm_output.asnumpy(), b_np,
+                                    rtol=1e-5, atol=1e-5)
+
+    test_apply(relay.exp, "fast_exp", np.exp, low=-88, high=88, step=0.01)
+    test_apply(relay.tanh, "fast_tanh", np.tanh, low=-10, high=10, step=0.01)
+
+    if __name__ == "__main__":
```
Comment (icemelon9) [inline tests/python/relay/test_op_fast_math.py]: fix the indent

**Comment by selo1412** [comment]
@comaniac @icemelon9 
Thanks for your help.
And, what should I do next?

By the way, I find that 
tvm/topi/tests/python/test_topi_math.py
Maybe, test_apply(fast_tanh) is missing in test_fastmath().

**Comment by icemelon9** [comment]
Yes, could you also add fast_tanh in the test_fastmath?

**Comment by icemelon9** [comment]
Thanks @selo1412 @comaniac. This is now merged.

**Pull request merged**